package.json

```diff
@@ -19,54 +19,54 @@
     "build-storybook": "build-storybook -s public"
   },
   "dependencies": {
+    "@svgr/cli": "^2.0.0",
```

Comment:
@hacdias is `@svgr/cli` the new `svgr`? 

There's a bunch of unrelated version bumps in here. In general please keep your PR changesets focused; they should contain just the changes required for the issue you are working on. 

You don't need to revert these version bumps, it's just a note for future PRs.